control-plane/config/eventing-kafka-broker/200-controller/100-config-kafka-leader-election.yaml

```diff
@@ -45,3 +45,5 @@ data:
   leaseDuration: "15s"
   renewDeadline: "10s"
   retryPeriod: "2s"
+  map-lease-prefix.kafka-broker-controller.knative.dev.eventing-kafka-broker.control-plane.pkg.reconciler.source.Reconciler: kafka-controller.knative.dev.eventing-kafka.pkg.source.reconciler.source.reconciler
```

Comment:
I am not questioning the setting, more `kafka-broker-controller` -> `kafka-controller` ?

Reply:
Yeah, changing that would require migration planning for existing Kafka brokers and KafkaSinks, so I'd keep it for now.